src/components/VideoPlayer/VideoPlayer.test.tsx

```diff
@@ -63,6 +63,25 @@ describe("VideoPlayer", () => {
     });
   });
 
+  describe("using embedCode prop", () => {
```

Comment:
Can you add a test where there is a custom `iframeTitle` prop and then we expect the iframe to have that new "title" attribute set? or just the `getByTitle` is fine as well.